icu4c/source/test/intltest/usettest.cpp

```diff
@@ -3984,3 +3985,31 @@ void UnicodeSetTest::TestDeepPattern() {
     assertTrue("[a[a[a...1000s...]]] -> error", errorCode.isFailure());
     errorCode.reset();
 }
+
+void UnicodeSetTest::TestEmptyString() {
+    IcuTestErrorCode errorCode(*this, "TestEmptyString");
+    // Starting with ICU 69, the empty string is allowed in UnicodeSet. ICU-13702
+    UnicodeSet set(u"[{}]", errorCode);
+    if (!assertSuccess("set from pattern with {}", errorCode)) { return; }
+    assertTrue("set from pattern with {}", set.contains(u""));
+    assertFalse("remove empty string", set.remove(u"").contains(u""));
```

Comment:
`.remove()` mutates the set and returns a self-reference? I know that's how it works, but it's confusing that your assertion statement not only asserts a truth; it also does work.

Reply:
I rather like it that way: It shows both the function that changes the contents and immediately tests its outcome.
And it's nicely concise.